httpie/core.py

```diff
@@ -108,19 +108,23 @@ def program(args, env, log_error):
             responses = [final_response]
 
         for response in responses:
-
             if args.check_status or downloader:
-                has_branched('program', 4)
+                if args.check_status:
```

Comment:
Shouldn't downloader be checked in an if-statement?

Reply:
it is implicitly, in the inner if-statement (in the else, if args.check_status is false then downloader is true - perhaps i should make it more clear?)

Reply:
Ah I see, yeah it could be done more explicitly I guess, I think @bttner is working on this patch right now though.